samples/contrib/aws-samples/mnist-kmeans-sagemaker/README.md

```diff
@@ -1,11 +1,20 @@
 # MNIST Classification with KMeans
 
-The `mnist-classification-pipeline.py` sample runs a pipeline to train a classficiation model using Kmeans with MNIST dataset on SageMaker. This example was taken from an existing [SageMaker example](https://github.com/awslabs/amazon-sagemaker-examples/blob/master/sagemaker-python-sdk/1P_kmeans_highlevel/kmeans_mnist.ipynb) and modified to work with the Amazon SageMaker Components for Kubeflow Pipelines.
+This sample aims to showcase the usage of v1 components and the new v2 training job component together in a Kubeflow pipeline workflow. The `mnist-classification-pipeline.py` sample runs a pipeline to train a classficiation model using Kmeans with MNIST dataset on SageMaker. This example was taken from an existing [SageMaker example](https://github.com/awslabs/amazon-sagemaker-examples/blob/master/sagemaker-python-sdk/1P_kmeans_highlevel/kmeans_mnist.ipynb) and modified to work with the Amazon SageMaker Components for Kubeflow Pipelines.
 
 ## Prerequisites 
 
-### Setup K8s cluster and authentication
-Make sure you have the setup explained in this [README.md](https://github.com/kubeflow/pipelines/blob/master/samples/contrib/aws-samples/README.md)
+1. Make sure you meet all the configuration prerequisites *including the optional step* for using the v2 components from sagemaker/TrainingJob README (https://github.com/kubeflow/pipelines/tree/master/components/aws/sagemaker/TrainingJob). 
```

Comment:
this will create a lot of confusion for the customer. The instructions need to be such that the userflow starts from here - https://github.com/kubeflow/pipelines/tree/master/samples/contrib/aws-samples.

Lets hold on to this PR. I will come up with a suggestion